src/Components/Display.jsx

```diff
@@ -1,16 +1,36 @@
-import React, {useState} from 'react';
+import React from 'react';
+import "./DisplayCss.css";
+import "bootstrap/dist/css/bootstrap.css";
+import AnimatedNumber from 'react-animated-number';
 
 function Display(props) {
-    // const [display,setDisplay] = useState(0);
-
+    
+    
     return (
-        <section className="display">
-            This is the Display for the Counter.
-        </section>
+        <div  className="Display" >
+            <h1 className="tag" ><AnimatedNumber
+							style={{
+								transition: '0.5s ease-out',
+								transitionProperty: 'background-color, color'
+							}}
+							frameStyle={perc =>
+								perc === 100
+									? {}
+									: { backgroundColor: 'blue' }
+							}
+							stepPrecision={0}
+							value={props.displayCount}
```

Comment:
inline styles?  I recommend sticking with external style sheets, because it becomes harder to track down style changes, because of the CSS cascade.  in-line styling has a higher weighting, and is harder to override.  There are use cases, but by default, start with the external style sheets, unless you have a good reason to do inline styles.